src/server/github_handler.rs

```diff
@@ -57,6 +57,7 @@ pub struct GithubEventHandler {
 const MAX_CONCURRENT_MERGES: usize = 20;
 const MAX_CONCURRENT_VERSIONS: usize = 20;
 const MAX_CONCURRENT_FORCE_PUSH: usize = 20;
+const MAX_COMMITS_FOR_JIRA_CONSIDERATION: usize = 20;
```

Comment:
doh. i meant to revert this and put it in its own PR. Forgot to push that change out. :disappointed: